core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1014,12 +1015,12 @@ class ReplicaManager(val config: KafkaConfig,
       val delayedProduce = new DelayedProduce(timeoutMs, produceMetadata, this, responseCallback, delayedProduceLock)
 
       // create a list of (topic, partition) pairs to use as keys for this delayed produce operation
-      val producerRequestKeys = entriesPerPartition.keys.map(TopicPartitionOperationKey(_)).toSeq
+      val producerRequestKeys = entriesPerPartition.keys.map(new TopicPartitionOperationKey(_)).toSeq
```

Comment:
Here we continue to use `toSeq`, but in some other places, we use `toList`. Could we be more consistent?

Reply:
Is this comment addressed? I still see a mix of `toSeq` and `toList`.

Reply:
I missed that one. I think this was the last one. Fixed